doc/configcommands.dsv

```diff
@@ -99,8 +99,13 @@ ttrss-mode|[multi/single]|multi|Configures the mode in which Tiny Tiny RSS is us
 ttrss-password|<password>|""|Configures the password for use with Tiny Tiny RSS. Double quotes should be escaped, i.e. you should write +{backslash}"+ instead of +"+.|ttrss-password "here_goesAquote:{backslash}""
 ttrss-passwordfile|<path-to-file|""|A more secure alternative to the above, by storing your password elsewhere in your system.|ttrss-passwordfile "path-to-file"
 ttrss-url|<url>|""|Configures the URL where the Tiny Tiny RSS installation you want to use resides.|ttrss-url "http://example.com/ttrss/"
+ocnews-login|<username>|""|Sets the username for use with the ownCloud instance.|ocnews-login "user"
```

Comment:
Sorry, I didn't know that. English isn't my first language, so I tend to just copy what I see other people do.